users/tests/test_allauth.py

```diff
@@ -0,0 +1,211 @@
+# -*- coding: utf-8 -*-
+from __future__ import absolute_import, unicode_literals
+
+from importlib import import_module
+from requests.exceptions import HTTPError
+
+from django.conf import settings
+from django.contrib.auth.models import User
+from django.core import mail
+from django.test.client import RequestFactory
+from django.test.utils import override_settings
+from django.urls import reverse
+
+from allauth.account import app_settings as account_settings
+from allauth.account.adapter import get_adapter
+from allauth.account.models import EmailAddress, EmailConfirmation
+from allauth.account.signals import user_signed_up
+from allauth.socialaccount.models import SocialAccount, SocialToken
+from allauth.socialaccount.tests import OAuth2TestsMixin
+from allauth.tests import MockedResponse, TestCase, patch
+
+from allauth.socialaccount.providers.google.provider import GoogleProvider
+
+# from allauth.socialaccount import views
+# from allauth.socialaccount.providers.google import views
+
+
+@override_settings(
+    SOCIALACCOUNT_AUTO_SIGNUP=True,
+    ACCOUNT_SIGNUP_FORM_CLASS=None,
+    ACCOUNT_EMAIL_VERIFICATION=account_settings.EmailVerificationMethod.MANDATORY,
+)
+class GoogleTests(OAuth2TestsMixin, TestCase):
```

Comment:
we are using django-pytest we are not using TestCases
My intention was to receive ideas and work in django-pytest not to copy 
sorry mate

Reply:
ok so it's too much work for me, and I spent too much time on this PR, I'm closing this PR then

Reply:
You are correct, but i would advise to maintain testing standards.
Please rethink even in cost of less test coverage, to work with django-pytest standards

Reply:
> To be honest I don't find much purpose in not reusing parts of complex code for testing integration of Django with Google API without needing to write from a scratch method by method, this test uses preprepared objects to ease the test of the sign in with google and it has been written for the purpose of testing this feature, I do see value in making modifications from using test case to PyTest but further than that not sure why is it bad reusing code if it feets my purposes, If something I thought not doing everything is part of a developer's job, to save valuable developing time,

I am not sure it is not helping your knowledge and skills as SW dev.  

> so there's time for the more important stuff. That's my opinion but that's only one opinion after all.

You are correct, but i would advise to maintain testing standards.
Please rethink even in cost of less test coverage, to work with django-pytest standards